src/Pkg.jl

```diff
@@ -139,13 +139,16 @@ const update = API.up
 
 
 """
-    Pkg.test(; coverage::Bool=false)
-    Pkg.test(pkg::Union{String, Vector{String}; coverage::Bool=false)
-    Pkg.test(pkgs::Union{PackageSpec, Vector{PackageSpec}}; coverage::Bool=false)
+    Pkg.test(; coverage::Bool=false, julia_args::AbstractVector{<:AbstractString}=String[], test_args::AbstractVector{<:AbstractString}=String[])
```

Comment:
This looks a bit ugly, maybe rewrite it as 
```
    Pkg.test(; kwargs...)
```
etc and explain allowed kwargs in a bulleted list below

Reply:
Done.